SharePointFramework/ProjectExtensions/src/projectSetup/index.ts

```diff
@@ -276,13 +276,19 @@ export default class ProjectSetup extends BaseApplicationCustomizer<IProjectSetu
         data: {},
         level: LogLevel.Info
       })
+
+      const templateFileName = (await sp.web.select("Title", "AllProperties").expand("AllProperties").get())['AllProperties']['pptemplate'];
```

Comment:
Need to fix lint issues so the build runs:

![image](https://user-images.githubusercontent.com/7606007/115538886-a0204780-a29c-11eb-8da6-25635aac466f.png)